src/content/docs/tutorial-errors/scope-the-problem.mdx

```diff
@@ -0,0 +1,159 @@
+    to="https://one.newrelic.com/nr1-core?state=985d4005-ba90-a8c7-1da1-2af34539b03b"
+  />
+
+  <TechTile
+    name="Java agent"
+    icon="logo-java"
+    to="https://one.newrelic.com/nr1-core?state=80d18bcb-4919-1fcb-2b77-9406838eb916"
+  />
+
+  <TechTile
+    name=".NET agent"
+    icon="logo-dotnet"
+    to="https://one.newrelic.com/nr1-core?state=30e93090-6dfa-6b70-8e75-472f54414355"
+  />
+
+  <TechTile
+    name="Node.js agent"
+    icon="logo-nodejs"
+    to="https://one.newrelic.com/marketplace/install-data-source?state=be2e62fa-cc3b-c428-27c4-8d662c9e80a1"
+  />
+
+  <TechTile
+    name="PHP agent"
+    icon="logo-php"
+    to="https://one.newrelic.com/nr1-core?state=aa633b41-72d4-009c-3abf-55dcf64894fe"
+  />
+
+  <TechTile
+    name="Python agent"
+    icon="logo-python"
+    to="https://one.newrelic.com/nr1-core?state=20fda75b-58fb-a92a-f9e1-7b052035c6e8"
+  />
+
+  <TechTile
+    name="Ruby agent"
+    icon="logo-ruby"
+    to="https://one.newrelic.com/nr1-core?state=d69143ab-605c-579b-25bf-cc6e5fee5b80"
+  />
+
+</TechTileGrid>
+
+Once you've installed an agent, go to **[one.newrelic.com](https://one.newrelic.com/nr1-core?filters=(domain%3D'APM'ANDtype%3D'APPLICATION'))** and select your app. If you don't see much data just yet, step away for a while and let the agent gather real-time data as your application runs.
+
+* [Integrate your logs](/docs/logs/get-started/get-started-log-management/#integrate-logs). The APM agent provides some logging details with logs in context, but certain key attributes could be missing from your log details. We recommend that you invest some time in setting up our full logs solution to expedite the error analysis process.
+
+## Find the critical error [#critical-error]
+
+With your apps instrumented, New Relic can capture data about your services. This includes data about error occurrences in your app. 
+
+<Steps>
+    <Step>
+        ### Think about the end user
+
+Similar to our flood metaphor, it's likely that when an outage occurs, the alerting services only indicate a problem exists somewhere. When you first receive an alert, you won't have enough context to make a decision about where to start in your system. 
+
+For example, imagine two services that have begun alerting, one for logging in and one for searching inventory. You're only receiving reports that the search functionality is failing for end users, but the login function works fine. You know in this case that, while logging in is critical to your funnel, you should prioritize the service that directly affects your end user. You may believe that the log in function is more important, but it's critical to separate your beliefs from your observability practices. 
+
+This practice applies even if the end user hasn't reported anything. When you notice services failing, you can ask yourself these questions:
+
+* Is the end user experiencing a problem? 
+* How should their experience look?
+* What behavior are they currently experiencing?
```

Comment:
I don't know what principles the tutorials squad has been using, but I'd try to get more concrete with this.  Like pose those questions **_and then use an example_**:

The end user can't add to cart. 
They should be able to add to cart.
Through alerts, we're seeing that their `cart-service` or whatever is experiencing latency. Response times is pretty easy to use for examples and easy to explain.